packages/edit-site/src/components/post-edit/index.js

```diff
@@ -49,7 +49,8 @@ function PostEditForm( { postType, postId } ) {
 	);
 	const [ multiEdits, setMultiEdits ] = useState( {} );
 	const { editEntityRecord } = useDispatch( coreDataStore );
-	const { fields: _fields } = usePostFields();
+	// TODO: This needs to be dynamic and will be handled in a follow up with the `format` field for posts.
+	const { fields: _fields } = usePostFields( { postType: 'page' } );
```

Comment:
Why not pass postType here? I don't understand

Reply:
The main reason for now, is that if I remember correctly we only had the `quick edit` only for `pages`. I don't remember why though and that was the reason for now. I need to check this better..

Reply:
Sure, but we should still pass the postType argument here. Whether we have quick edit for pages or posts is controlled elsewhere. (Whether PostEdit is rendered)